### ISSUE TITLE
Duplicate key in log entry

### ISSUE THREAD

**Issue by emilbillberg** [VictoriaMetrics/operator#1104]
Hi, we are using victoria-metrics-operator version v0.47.3. We have an issue with the logs containing duplicate keys of different types which causes an error in our log backend. This is the log entry and as you can see it contains `vmalert` multiple times:

```
{"level":"info","ts":"2024-09-16T07:48:12Z","logger":"controller.VMAlert","msg":"no Rule changes","vmalert":{"name":"alert","namespace":"infra"},"reconcile":"rulesCm","vmalert":"alert","namespace":"infra"}
```

**Comment by f41gh7** [comment]
Issue will be fixed at the next release. Operator will consistently use the same type for logger keys.

**Comment by f41gh7** [comment]
Issue fixed at  [v0.48.3](https://github.com/VictoriaMetrics/operator/releases/tag/v0.48.3) release.
